app/models/submission.rb

```diff
@@ -392,25 +392,9 @@ def as_json(options = {})
   end
 
   def grading_complete?(as_seen_by)
```

Comment:
nit: name of the function was kind of confusing for me, maybe rename to `grading_released`

Reply:
Renamed to `grades_released?`